tests/git/test_git_commands.py

```diff
@@ -587,20 +589,29 @@ def test_execute(
 ):
     mock_execute.return_value = mock_cursor(
         [
-            {"name": "repo/branches/main/a/S3.sql"},
-            {"name": "repo/branches/main/s1.sql"},
-            {"name": "repo/branches/main/s2"},
+            {"name": "/a/S3.sql"},
+            {"name": "/s1.sql"},
+            {"name": "/s2"},
```

Comment:
Why you changed list-files response?

Reply:
Because the list is done on stage not on repo.